test/include/common/test_util.h

```diff
@@ -65,5 +67,46 @@ void InvokeWorkloadWithDistribution(std::vector<std::function<void()>> workloads
   for (uint32_t i = 0; i < repeat; i++)
     workloads[dist(generator)]();
 }
+
+#define TO_INT(p) reinterpret_cast<uintptr_t>(p)
+/**
+ * Check if val in [lower, upper)
```

Comment:
It would be more clear if you can note the function is for memory addresses.

Reply:
Input types are ptrs, but sure.